doc/articles/get-started-licensing.md

```diff
@@ -0,0 +1,72 @@
+
+1. Create a new Uno Platform project or open an existing one with `Uno.Sdk` version 5.5 or higher.
+2. After opening and loading completely your project, a popup should appear. Click **Sign in / Register**.
+   ![Visual Studio 2022 Popup](Assets/uno-settings-popup.png)
+   >![NOTE]
+   > If the popup doesn’t appear, access Uno Platform settings by clicking on **Extensions** > **Uno Platform** > **Settings...**.
+   > ![Visual Studio 2022 Menu](Assets/uno-settings-vs.png)
+
+### [**Visual Studio Code**](#tab/vscode)
+
+If you’ve already set up **Visual Studio Code** by following [Get Started on VS Code documentation](xref:Uno.GettingStarted.CreateAnApp.VSCode), sign in as follows:
+
+1. Create a new Uno Platform project or open an existing one with `Uno.Sdk` version 5.5 or higher.
+2. After opening and loading completely your project, click the red Solution Button in the bottom status bar and switch to `.csproj` instead of `.sln`.
+   ![VS Code Solution Button](Assets/uno-vsc-csproj.gif)
+3. After opening and loading completely your project, a popup should appear. Click **Sign in / Register**.
+   ![Visual Studio Code Menu](Assets/uno-settings-vsc-popup.png)
+   >![NOTE]
+   > If the popup doesn’t appear, access **Uno Platform** extension settings by selecting **View** > **Command Palette...** and typing `Uno Platform: Open Settings`.
+   > ![Visual Studio Code Menu](Assets/uno-settings-vsc.png)
+
+### [**JetBrains Rider**](#tab/rider)
+
+If you’ve already set up **JetBrains Rider** by following [Get Started on JetBrains Rider documentation](xref:Uno.GetStarted.Rider), sign in as follows:
+
+1. Create a new Uno Platform project or open an existing one with `Uno.Sdk` version 5.5 or higher.
+2. After opening and loading completely your project, a popup should appear. Click **Sign in / Register**.
+   ![JetBrains Rider Popup](Assets/uno-settings-rider-popup.png)
+   >![NOTE]
+   > If the popup doesn’t appear, access **Uno Platform** extension settings by selecting **Tools** > **Uno Platform** > **Settings...**.
+   > ![JetBrains Rider Menu](Assets/uno-settings-rider.png)
+
+---
+
+1. In the Uno Platform Settings window, click on **Sign in**. You’ll be redirected to your browser to enter your Uno Platform account credentials.
+   ![Uno Platform Settings Welcome](Assets/uno-settings-welcome.png)
+2. Once signed in, you’ll see a confirmation of your account along with your license details. You can then use the **Hot Reload** feature to speed up your workflow and test changes in real time. For more information, refer to the [Hot Reload documentation](xref:Uno.Features.HotReload).
+   ![Uno Platform Settings](Assets/uno-settings-main.png)
+   >![TIP]
+   > You can also access a menu where you can select **My Account** to view your account details, **Refresh** the account changes and **Sign out**.
+   > ![Uno Platform Menu](Assets/uno-settings-menu.png)
```

Comment:
```suggestion
1. In the Uno Platform Settings window, click on **Sign in**. You’ll be redirected to your browser to enter your Uno Platform account credentials.

![Uno Platform Settings Welcome](Assets/uno-settings-welcome.png)
2. Once signed in, you’ll see a confirmation of your account along with your license details. You can then use the **Hot Reload** feature to speed up your workflow and test changes in real time. For more information, refer to the [Hot Reload documentation](xref:Uno.Features.HotReload).

   ![Uno Platform Settings](Assets/uno-settings-main.png)
   >![TIP]
   > You can also access a menu where you can select **My Account** to view your account details, **Refresh** the account changes and **Sign out**.
   >
   > ![Uno Platform Menu](Assets/uno-settings-menu.png)
```

Reply:
Same thing here for the images, to prevent this:

![image](https://github.com/user-attachments/assets/76288ad5-0bc1-4499-83b4-9b884f13c5cf)